src/index.js

```diff
@@ -1,26 +1,25 @@
-import { leaderboard, UpdateLocalStorage } from './modules/data.js';
+// import { leaderboard } from './modules/data.js';
 import {
   addBtn, nameIn, refreshBtn, scoreIn,
 } from './modules/domlist.js';
-import writeList from './modules/innerhtml.js';
 import Leader from './modules/leaders.js';
+import refreshList from './modules/refresh-api.js';
+import addLeader from './modules/post-api.js';
 import './style.css';
 
-writeList();
+// setInterval(() => {
+//   console.log(leaderboard[0][0]);
+// },3000);
```

Comment:
- [ ] Please kindly remove `commented code` from your `code base`, as it makes your `code base` dirty and less readable. To make sure that your codebase is always clean, always remember to remove commented-codes.